test/integration/submit_pledge_test.rb

```diff
@@ -133,5 +133,32 @@ class SubmitPledgeTest < ActionDispatch::IntegrationTest
 
       assert has_link? 'Submit pledge'
     end
+
+    it 'should dynamically update page' do
```

Comment:
I'd like to spin this out into its own test since it's basically a feature in its own right $$